src/Map/src/Bridge/Google/src/Option/ControlPosition.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google\Option;
+
+/**
+ * Identifiers used to specify the placement of controls on the map.
+ * Controls are positioned relative to other controls in the same layout position.
+ * Controls that are added first are positioned closer to the edge of the map.
+ * Usage of "logical values" (see https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) is recommended
+ * in order to be able to automatically support both left-to-right (LTR) and right-to-left (RTL) layout contexts.
+ *
+ * @see https://developers.google.com/maps/documentation/javascript/reference/control#ControlPosition
+ *
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+enum ControlPosition: int
+{
+    /**
+     * Equivalent to {@see self::BOTTOM_CENTER} in both LTR and RTL.
+     */
+    case BLOCK_END_INLINE_CENTER = 24;
+
+    /**
+     * Equivalent to {@see self::BOTTOM_RIGHT} in LTR, or {@see self::BOTTOM_LEFT} in RTL.
+     */
+    case BLOCK_END_INLINE_END = 25;
+
+    /**
+     * Equivalent to {@see self::BOTTOM_LEFT} in LTR, or {@see self::BOTTOM_RIGHT} in RTL.
+     */
+    case BLOCK_END_INLINE_START = 23;
+
+    /**
+     * Equivalent to {@see self::TOP_CENTER} in both LTR and RTL.
+     */
+    case BLOCK_START_INLINE_CENTER = 15;
+
+    /**
+     * Equivalent to {@see self::TOP_RIGHT} in LTR, or {@see self::TOP_LEFT} in RTL.
+     */
+    case BLOCK_START_INLINE_END = 16;
+
+    /**
+     * Equivalent to {@see self::TOP_LEFT} in LTR, or {@see self::TOP_RIGHT} in RTL.
+     */
+    case BLOCK_START_INLINE_START = 14;
+
+    /**
+     * Equivalent to {@see self::RIGHT_CENTER} in LTR, or {@see self::LEFT_CENTER} in RTL.
+     */
+    case INLINE_END_BLOCK_CENTER = 21;
+
+    /**
+     * Equivalent to {@see self::RIGHT_BOTTOM} in LTR, or {@see self::LEFT_BOTTOM} in RTL.
+     */
+    case INLINE_END_BLOCK_END = 22;
+
+    /**
+     * Equivalent to {@see self::RIGHT_TOP} in LTR, or {@see self::LEFT_TOP} in RTL.
+     */
+    case INLINE_END_BLOCK_START = 20;
+
+    /**
+     * Equivalent to {@see self::LEFT_CENTER} in LTR, or {@see self::RIGHT_CENTER} in RTL.
+     */
+    case INLINE_START_BLOCK_CENTER = 17;
+
+    /**
+     * Equivalent to {@see self::LEFT_BOTTOM} in LTR, or {@see self::RIGHT_BOTTOM} in RTL.
+     */
+    case INLINE_START_BLOCK_END = 19;
+
+    /**
+     * Equivalent to {@see self::LEFT_TOP} in LTR, or {@see self::RIGHT_TOP} in RTL.
+     */
```

Comment:
You can remove all the equivalent docblocks :)

Reply:
Why? What are the advantages of removing documentation?

Reply:
you don't have LEFT_TOP anymore :)

Reply:
Ah, yes you're right. 

But reading `BLOCK_END_INLINE_CENTER` alone does not help to understand what it does, I will keep the documentation but replace `{@see self::...}` by their humanized equivalent.